pkg/agent/openflow/client.go

```diff
@@ -793,28 +793,46 @@ func (c *client) GetServiceFlowKeys(svcIP net.IP, svcPort uint16, protocol bindi
 }
 
 func (c *client) initialize() error {
+	// After a connection or re-connection, delete all existing group and meter entries, to
+	// avoid "already exist" errors. This will typically happen if the antrea-agent container is
+	// restarted (but not the antrea-ovs one). We do this in initialize(), and not directly in
+	// Initialize(), to ensure that the deletion happen on every re-connection (when
+	// ReplayFlows() is called), even though we typically only see reconnections when the OVS
+	// daemons are restarted. When ovs-vswitchd restarts, group and meter entries are empty by
+	// default and these calls are not required.
+	// This is specific to groups and meters. Flows are replayed with a different cookie number
+	// and conflicts are not possible.
+	if err := c.bridge.DeleteGroupAll(); err != nil {
+		return fmt.Errorf("error when deleting all group entries: %w", err)
+	}
+	if c.ovsMetersAreSupported {
+		if err := c.bridge.DeleteMeterAll(); err != nil {
```

Comment:
So actually there was a bug that if antrea-agent reconnects to OVS (without any process restarted), replaying flows would fail when adding meters?

Reply:
I am not sure I would call it a bug. It's not clear to me whether we can have an actual disconnect / reconnect over the UDS without either of the processes / containers dying and restarting. At least I don't think a user has ever reported an issue about this.

But yes, if such a scenario could happen, then replaying the flow would fail with `OFPMMFC_METER_EXISTS`.

Essentially in this PR I kept the behavior the same for groups (I am just moving the responsibility from ofnet to antrea), and I made the treatment of meters consistent with groups (deletion happens on every reconnect / reply, not just during the first agent initialization).

I'll wait for a review from @wenyingd, to get an extra pair of eyes on this. This is also meant for 1.15 (not 1.14) and I don't intend to backport.